tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -75,6 +75,40 @@ class WskBasicTests
             stderr should include(errormsg)
     }
 
+    it should "accept a 3 part Fully Qualified Name without a leading '/'" in {
+        val auth: Seq[String] = Seq("--auth", wskprops.authKey)
+        val guestNamespace = wskprops.namespace
+        val packageName = "packageName3ptFQN"
+        val actionName = "actionName3ptFQN"
+        val triggerName = "triggerName3ptFQN"
+        val ruleName = "ruleName3ptFQN"
+        val fullQualifiedName = s"${guestNamespace}/${packageName}/${actionName}"
+        val validArgs: Seq[Seq[String]] = Seq(
+            Seq("package", "create", packageName),
+            Seq("action", "create", fullQualifiedName, defaultAction.get),
+            Seq("action", "update", fullQualifiedName, defaultAction.get),
+            Seq("action", "invoke", fullQualifiedName),
+            Seq("trigger", "create", triggerName),
```

Comment:
why are these here? they don't test anything not already tested (triggers and rules are 2 parts only and so is a package name).